optionals/compat_sog/script_component.hpp

```diff
@@ -7,3 +7,4 @@
 
 #define EXPLOSIVES_PLACE(CLASS) EGVAR(explosives,DOUBLES(Place,CLASS))
 #define QEXPLOSIVES_PLACE(CLASS) QUOTE(EXPLOSIVES_PLACE(CLASS))
+#define REFUEL_INFINITE_FUEL -10
```

Comment:
Wouldn't it be cleaner and better for maintenance to include the `script_components` file from the refuel module, instead of introducing a duplicate macro?

Reply:
It would. @commy2 sorry for the ping. Can you add his suggestion? Not going to be near a computer till tomorrow.

Reply:
Nvm, redefines component, produces errors in build